dashboard/src/components/AppView/AppControls/RollbackButton/RollbackDialog.tsx

```diff
@@ -37,13 +37,15 @@ class RollbackDialog extends React.Component<IRollbackDialogProps, IRollbackDial
         <label>
           Select the revision to which you want to rollback (current: {this.props.currentRevision})
         </label>
-        <select className="margin-t-normal" onChange={this.selectRevision}>
-          {options.map(o => (
-            <option key={o} value={o}>
-              {o}
-            </option>
-          ))}
-        </select>
+        <div>
```

Comment:
I have moved the `select` block (inline) to its own div because `label` and `select` are inline components. That caused the modal to have the width of `select+label` but they were rendered in different lines, making the modal unnecessarily big.

Reply:
I hate fiddling with CSS like that. Angel will probably comment when back with some sage advice, I'm sure :)